src/main/java/org/molgenis/validators/GeneValidationService.java

```diff
@@ -0,0 +1,89 @@
+package org.molgenis.validators;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.camel.Exchange;
+import org.molgenis.utils.InvalidGeneException;
+
+public class GeneValidationService {
+
+  private static final String STATUS = "status";
+  private static final String APPROVED = "approved";
+  private static final String ERROR = "error";
+
+  Map<String, HashMap<String, String>> genes;
+  Map<String, String> geneAlternatives;
```

Comment:
`Alternative` is the same as `Alias gene symbol`? Then I would go for the letter. Is the key a gene identifier or the approved gene symbol?

Reply:
No, the alternatives is the alias and previous symbols combined.